src/Compilation.zig

```diff
@@ -3292,10 +3295,26 @@ pub fn getAllErrorsAlloc(comp: *Compilation) !ErrorBundle {
     return bundle.toOwnedBundle(compile_log_text);
 }
 
-fn totalErrorCount(comp: *Compilation) !u32 {
-    var errors = try comp.getAllErrorsAlloc();
-    defer errors.deinit(comp.gpa);
-    return errors.errorMessageCount();
+fn anyErrors(comp: *Compilation) bool {
+    const zcu_has_errors = zcu_has_errors: {
+        const zcu = comp.zcu orelse break :zcu_has_errors false;
+        break :zcu_has_errors zcu.failed_files.values().len > 0 or
+            zcu.failed_embed_files.values().len > 0 or
+            zcu.failed_analysis.values().len > 0 or
+            zcu.failed_codegen.values().len > 0 or
+            zcu.failed_exports.values().len > 0 or
+            zcu.compile_log_sources.count() > 0 or
+            (comp.incremental and zcu.transitive_failed_analysis.values().len > 0);
+    };
+    return comp.failed_c_objects.values().len > 0 or
+        comp.failed_win32_resources.values().len > 0 or
+        comp.lld_errors.items.len > 0 or
+        comp.misc_failures.values().len > 0 or
+        comp.alloc_failure_occurred or
+        comp.link_error_flags.missing_libc or
+        comp.link_errors.items.len > 0 or
+        comp.link_error_flags.no_entry_point_found or
```

Comment:
should instead bit cast to int and check for nonzero (this would fix the bug here which is not noticing the missing_libc field)